pkg/commands/apono/runner.go

```diff
@@ -129,6 +130,7 @@ func createRootCommand(versionInfo version.VersionInfo) *cobra.Command {
 		cmd.SetContext(version.CreateVersionContext(cmd.Context(), &versionInfo))
 		cmd.SetContext(analytics.CreateStartTimeContext(cmd.Context(), &commandStartTime))
 		cmd.SetContext(analytics.CreateCommandIDContext(cmd.Context(), commandID))
+		cmd.SetContext(config.CreateProfileContext(cmd.Context(), profileName))
```

Comment:
what is this for?

Reply:
i need to command current profile in the state for getting the Portal URL